french/script/story/st_00.rpy

```diff
@@ -206,103 +206,103 @@ translate french s00_birth_of_the_red_fang_1371ab12:
 translate french s00_birth_of_the_red_fang_bc140428:
 
     # sk sun angry "And you would?{w=0.3} Pray tell, then, how are we going to win a race war outnumbered five to one?!"
-    sk sun angry ""
+    sk sun angry "Et le ferais-tu ?{w=0.3} Je t'en prie, dis-le, alors, comment allons nous gagner une guerre raciale à cinq contre un ?!"
```

Comment:
pray tell c'est "dis-moi" en français, parce que l'objet "me" est sous entendu en anglais.

Je pense que c'est plus fluide si tu inverse "alors" et "dis-le/dis-moi" en fr, "je t'en prie, alors, dis-moi comment on va gagner une guerre raciale à cinq contre un?" (j'ai mis on et pas nous vu que c'est un peu trop formel dans le contexte où ils vont en venir aux armes bientôt, ça me semble être la phrase parfaite pour que Sienna perde sa politesse)

le "and you would" est un défi, que Sienna lance à Adam (il pense qu'il peux le faire, c'est un peu notre "tu penses vraiment que tu aurais fait mieux?"), ta traduction est correcte mais l'intention n'est pas traduite.
J'ai pensé à "Et tu le pourrais?" mais c'est un peu plat, par contre "Et tu le pourrais, toi?" je trouve que l'idée serait entièrement traduite.

(à partir de mtn, je te mets direct la phrase avec mes suggestions appliqués, ça ira plus vite pour voir le résultat):
"Et tu le pourrais, toi ?{w=0.3} Je t'en prie, alors, dis-moi, comment on va gagner une guerre raciale à cinq contre un ?!"

Reply:
Ca marche !